Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -566,6 +566,7 @@ CSS::RequiredInvalidationAfterStyleChange Element::recompute_style()
 
     recompute_pseudo_element_style(CSS::PseudoElement::Before);
     recompute_pseudo_element_style(CSS::PseudoElement::After);
+    recompute_pseudo_element_style(CSS::PseudoElement::Backdrop);
```

Comment:
Ideally we'd guard this like we do for `::marker` just below, to only compute it when needed.